README.md

```diff
@@ -49,25 +50,64 @@ The purpose of this meeting is to provide in depth support and guidance to the c
 ##### Details
 
   * Date/Time: [Thursdays at 2pm ET (18:00 UTC)](https://buildpacks.io/community/#calendar)
-  * Location: [Zoom](https://VMware.zoom.us/j/95751108957?pwd=Vlg5dndyL1pYeUxPck1BL2ZSMjQ1UT09)
+  * Location: [Zoom](https://zoom.us/j/93521338058?pwd=eDl0R3IyenY5b2FzS2Nwa25taFNOQT09)
   * Meeting Notes: [Google Doc](https://docs.google.com/document/d/1CwixRknj3gAXbi-fTI1-chwtMfy6G3GhdH17aEUjhK0/edit)
   * Video Archives: [YouTube](https://www.youtube.com/playlist?list=PL1p8pquzNvRqnn4R_eOoDxVve2HzY8o0n)
+  * Meeting Invite: [ICS](https://zoom.us/meeting/tJcodeirrTMqH9yIgEiIeXqpsUaJAaBFr0Ra/ics?icsToken=98tyKuCrqzkrGdedsByPRowqGYjod-nztlxfgvpEtRjACglWWhLmJddDGJN-Mv7C)
```

Comment:
Do we want to use Zoom's ICS or TeamUps? I have an inclination to vote for TeamUp so that there's only one place to manage them. For example, in the case of a cancellation. If we want ICS per sub-team we can structure it that way in TeamUp.